.travis.yml

```diff
@@ -24,7 +24,7 @@ script:
   - cd docs; make clean; make html
   # Run the test suit with coverage
   - cd ..
-  - travis_wait 60 coverage run --source=axelrod -m unittest discover
+  - travis_wait 60 coverage run --source=axelrod -m unittest discover -v
```

Comment:
Double checking that the verbose tag isn't just something you wanted for debugging @marcharper? I don't necessarily mind either way (it does make scrolling through the travis log a big long but no big deal).

Reply:
I'll take it out

Reply:
I'm fine with adding it back in, I did add it just for debugging Travis